src/layers/medCore/legacy/data/medAbstractData.h

```diff
@@ -58,7 +61,11 @@ class MEDCORE_EXPORT medAbstractData : public dtkAbstractData
     QList<medAbstractData*> derivedData();
     void setExpectedName(QString name);
     QString getExpectedName();
+	
+    QString fecthMetaData(QString word) const;
+    QMap<QString, QString> getMetaDataMap() const;
 
+    //virtual QByteArray serialise() const = 0 ;
```

Comment:
i don't know why my comment disappeared, maybe because of an update of this file.
There are empty lines , this commented line, and the commented line 55 to remove. And the "fecthMetaData" to correct in all this PR too

Reply:
I want to keep this comment because it's my next work.